scripts/abins/kpointsdata.py

```diff
@@ -129,6 +130,17 @@ def get_gamma_point_data(self):
 
         return k_points
 
+    def is_normalised(self) -> bool:
+        """
+        Check atomic displacements are normalised correctly
+        """
+        for displacements in self._atomic_displacements:
+            if not np.allclose(np.ones(displacements.shape[1]),
+                               norm(norm(displacements, axis=0), axis=1)):
+                return False
+        else:
```

Comment:
this else does not match up with anything

Reply:
It's valid Python, for-else is used for a statement that should only run if the loop is not otherwise broken. https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops

It should really be used with `break` rather then `return`, though: in thise case it can be removed.